airbyte-ci/connectors/pipelines/pipelines/actions/environments.py

```diff
@@ -678,30 +623,47 @@ def with_integration_base(context: PipelineContext, build_platform: Platform) ->
     )
 
 
-def with_integration_base_java(context: PipelineContext, build_platform: Platform, jdk_version: str = "17.0.4") -> Container:
+def with_integration_base_java(context: PipelineContext, build_platform: Platform) -> Container:
```

Comment:
FYI this might be extracted from this repo  soon to use our upcoming framework to define connector base image.
It's in active development for Python: 
https://github.com/airbytehq/airbyte/pull/30303